managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/ManagedCursorTest.java

```diff
@@ -3526,5 +3528,26 @@ public void testCursorCheckReadPositionChanged() throws Exception {
         ledger.close();
     }
 
+
+    @Test // (timeOut = 20000)
+    public void testCursorGetBacklog() throws Exception {
+        ManagedLedgerConfig managedLedgerConfig = new ManagedLedgerConfig();
+        managedLedgerConfig.setMaxEntriesPerLedger(1);
```

Comment:
Do we need test the scenario that one ledger maintain multiple entries?

Reply:
good idea.